.travis/providers/aws-elasticbeanstalk/travis.yml

```diff
@@ -20,6 +20,7 @@ deploy:
     app: dpl-test
     env: DplTest-env
     bucket_name: travis-dpl-test-elasticbeanstalk
+    debug: true
```

Comment:
Is this needed now?

Reply:
yes, it makes the provider output the file names/paths added to the zip archive. i don't think we want to do this by default, and i've made the option `internal`, so it won't show up in the readme/help output